index.html

```diff
@@ -67,11 +66,167 @@ <h3>
+            <div class="flex-post">
+              <ul>
+                <li><a href="#">Ruby on rails</a></li>
+                <li><a href="#">CSS</a></li>
+                <li><a href="#">Javascript</a></li>
+                <li><a href="#">HTML</a></li>
+              </ul>
+            </div>
+            <button type="button" class="works-button">See Project</button>
+          </div>
+        </div>
+        <div class="work-6">
+          <div class="img-holder"></div>
+          <div class="posts">
+            <h2>Multi-Post Stories Gain+Glory</h2>
+            <div class="flex-post">
+              <ul>
+                <li><a href="#">Ruby on rails</a></li>
+                <li><a href="#">CSS</a></li>
+                <li><a href="#">Javascript</a></li>
+                <li><a href="#">HTML</a></li>
+              </ul>
+            </div>
+            <button type="button" class="works-button">See Project</button>
+          </div>
+        </div>
+      </div>
+    </div>
+    <section class="about-me">
+      <div class="about-me-text">
+        <h2>About Me</h2>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <button type="button" class="get-resume">Get My Resume</button>
+      </div>
+    </section>
+    <div class="my-knowledge">
+      <section class="languages">
+        <img src="images/lang.svg" alt="languages" />
+        <h2>Languages</h2>
+        <div class="flex-languages">
+          <ul>
+            <li><a href="#">JavaScript</a></li>
+            <li><a href="#">Ruby</a></li>
+            <li><a href="#">HTML</a></li>
+            <li><a href="#">CSS</a></li>
+          </ul>
+        </div>
+      </section>
+      <section class="frameworks">
+        <img src="images/frameworks.svg" alt="frameworks" />
+        <h2>Frameworks</h2>
+        <div class="flex-frameworks">
+          <ul>
+            <li><a href="#">Bootstrap</a></li>
+            <li><a href="#">Ruby on Rails</a></li>
+            <li><a href="#">RSpec</a></li>
+            <li><a href="#">Copybara</a></li>
+            <li><a href="#">Selenium</a></li>
+          </ul>
+        </div>
+      </section>
+      <section class="skills">
+        <img src="images/skills.svg" alt="skills" />
+        <h2>Skills</h2>
+        <div class="flex-skills">
+          <ul>
+            <li><a href="#">Codekit</a></li>
+            <li><a href="#">Github</a></li>
+            <li><a href="#">Codepen</a></li>
+            <li><a href="#">Gitlab</a></li>
+            <li><a href="#">Terminal</a></li>
+          </ul>
+        </div>
+      </section>
+    </div>
+    <div class="contact-me">
+      <div class="contact-me-content">
+        <h3>
+          I'm always interested in hearing about new projects, so if you'd like
```

Comment:
- [ ] Kindly decrease the font size of the heading to be exactly like [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468).

Here you have a screenshot from your project:

![image](https://user-images.githubusercontent.com/74506933/159126941-3519062f-2168-41d6-b950-cffe89032233.png)

Here you have a screenshot from the design:

![image](https://user-images.githubusercontent.com/74506933/159126963-29d3f509-86b0-4cd1-83c1-ac3452e2ea38.png)